src/components/KTooltip/KTooltip.vue

```diff
@@ -72,63 +71,69 @@ const props = defineProps({
     type: String,
     default: 'auto',
   },
+
   /**
-  * Test mode - for testing only, strips out generated ids
-  */
-  testMode: {
-    type: Boolean,
-    default: false,
+   * @deprecated in favor of text prop
+   */
+  label: {
+    type: String,
+    default: '',
   },
 })
 
 const slots = useSlots()
-const showTooltip = computed((): boolean => !!props.label || !!slots.content)
+const showTooltip = computed((): boolean => !!props.text || !!slots.content || !!props.label)
```

Comment:
Shouldn't the label prop go before the slot, just like the text prop?

Reply:
I put it in last because `label` is deprecated so more relevant ones come first

Reply:
regardless, should the slot go before the prop? Is our current pattern to give preference to slots?

Reply:
I don't really care so changed it to what you suggest